<launch>

   <node name="robotino_node" pkg="cnbiros_robotino" type="robotino_node" output="screen" >
      <param name="hostname" value="192.168.1.3" />
      <param name="topic_velocity" value="/robotino_velocity" />
      <param name="topic_infrared" value="/robotino_infrared" />
   </node>
   
</launch>
